python/lsst/rubintv/production/pipelineRunning.py

```diff
@@ -113,24 +124,30 @@ def __init__(
             # in some contexts but not all. Maybe default it to
             # ''?
             channelName="",
-            queueName=queueName,
+            podDetails=podDetails,
             doRaise=doRaise,
+            addUploader=False,  # pipeline running pods don't upload directly
+        )
+        self.instrument: str = instrument
+        self.butler: Butler = butler
+        self.step: str = step
+        self.pipeline: str = pipeline + f"#{step}"
+        self.pipelineGraph: PipelineGraph = Pipeline.fromFile(self.pipeline).to_graph(
+            registry=self.butler.registry
         )
-        self.instrument = instrument
-        self.butler = butler
-        self.step = step
-        self.pipeline = pipeline + f"#{step}"
-        self.pipelineGraph = Pipeline.fromFile(self.pipeline).to_graph(registry=self.butler.registry)
-        self.pipelineGraphBytes = pipelineGraphToBytes(self.pipelineGraph)
+        self.pipelineGraphBytes: bytes = pipelineGraphToBytes(self.pipelineGraph)
+        self.podDetails: PodDetails = podDetails
 
-        self.runCollection = None
-        self.limitedButler = self.makeLimitedButler(butler)
-        self.log.info(f"Pipeline running configured to consume from {queueName}")
+        self.runCollection: str | None = None
+        self.limitedButler: CachingLimitedButler = self.makeLimitedButler(butler)
+        self.log.info(f"Pipeline running configured to consume from {self.podDetails.queueName}")
 
-        self.consdbClient = ConsDbClient("http://consdb-pq.consdb:8080/consdb")
-        self.consDBPopulator = ConsDBPopulator(self.consdbClient, self.watcher.redisHelper)
+        self.consdbClient: ConsDbClient = ConsDbClient("http://consdb-pq.consdb:8080/consdb")
```

Comment:
Yeah, interesting. This is only supposed to run at the summit, and I think that's pretty static, but I did indeed consider shoe-horning it into the already-awful `LocationConfig` because that's currently where all the config stuff lives. I think my hand will be forced there once the NV tables exist and start being populated at USDF, but I'm not holding my breath there!